packit_service/constants.py

```diff
@@ -236,11 +236,12 @@ def from_number(number: int):
     "Not allowed to build in {copr_project} Copr project."
 )
 CUSTOM_COPR_PROJECT_NOT_ALLOWED_CONTENT = (
-    "Your git-forge project is not allowed to use the configured `{copr_project}` Copr project.\n\n"
-    f"Please, [let us know]({CONTACTS_URL}) "
-    "if you need this git project to be allowed. "
-    "We are working with the Copr team on a way "
-    "how to make this easily configurable in the Copr web interface.\n"
+    "Your git-forge project is not allowed to use "
+    "the configured `{copr_project}` Copr project.\n\n"
+    "Please, add this git-forge project {forge_project} "
+    'to the list named "Packit allowed forge projects" '
+    "in the copr project page settings: "
+    "https://copr.fedorainfracloud.org/coprs/{copr_project}/edit/."
```

Comment:
```suggestion
    "Please, add this git-forge project `{forge_project}` "
    'to `Packit allowed forge projects`'
    "in the [Copr project settings](https://copr.fedorainfracloud.org/coprs/{copr_project}/edit/#packit_forge_projects_allowed): "
```
we can navigate directly to the field